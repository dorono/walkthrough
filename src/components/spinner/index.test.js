import React from 'react';
import {shallow} from 'enzyme';
import Spinner from './index';

describe('Spinner', () => {
    it('should render without errors', () => {
        shallow(<Spinner />);
    });

    it('should match snapshot', () => {
        expect(shallow(<Spinner />)).toMatchSnapshot();
    });
});
