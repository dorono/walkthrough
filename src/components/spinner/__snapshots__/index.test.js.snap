// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner should match snapshot 1`] = `
<div
  className="root"
>
  <div
    className="rect1"
  />
  <div
    className="rect2"
  />
  <div
    className="rect3"
  />
  <div
    className="rect4"
  />
  <div
    className="rect5"
  />
</div>
`;
